clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/secured/ConfigurationUtils.java

```diff
@@ -234,15 +237,15 @@ public <T> T get(String name) {
         return (T) configs.get(name);
     }
 
+    // visible for testing
     // make sure the url is in the "org.apache.kafka.sasl.oauthbearer.allowed.urls" system property
-    public void throwIfURLIsNotAllowed(String urlConfig) {
-        Set<String> allowedLoginModuleList = Arrays.stream(
+    void throwIfURLIsNotAllowed(String value) {
+        Set<String> allowedUrlsList = Arrays.stream(
```

Comment:
nit: it's a `Set` so not sure why we can it `List`, it could just be `allowedUrls`.

Reply:
Ah, right! Updated!